app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Here is a list of all posts by user 1</h1>
\ No newline at end of file
+<article class="main-article" style="width: 90%; margin: auto">
+  <div style="display: flex; align-items: center; justify-content: space-between;">
+    <%= image_tag @user.photo, :class => "user-avatar" %>
+    <div class="user-details">
+      <p><%= @user.name %></p>
+      <span>Number of posts: <%= @user.post_counter %></span>
+    </div>
+  </div>
+
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div>
+      <div class="post">
+        <h4>Post #<%= index + 1 %></h4>
+        <div>
+          <p class="post-title"><%= post.title %></p>
+        </div>
+        <div class="post-footer">
+          <span>Comments: <%= post.comments_counter %></span>
+          <span>Likes: <%= post.likes_counter %></span>
+        </div>
+      </div>
+      <div class="post-comments">
+        <div class="comments">
+          <% post.recent_comments.each do |comment| %>
+            <div class="comment">
+              <span>User <%= comment.user_id %></span> : <span><%= comment.text %></span>
+            </div>
+          <% end %>
+        </div>
+      </div>
+    </div>
+  <% end %>
```

Comment:
- [ ] [Optional] Well! currently, on the Posts#index page, you are displaying only the most recent 3 posts for a specific user. However, as per the wireframe, you should display all posts for a specific user. Additionally, if necessary, you should consider adding pagination. I recommend following the project requirements and updating your code accordingly.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/simplegoose/blog_app/assets/55840999/313bcb2d-400d-40a5-b5aa-5b35a6d85acc)| ![image](https://github.com/simplegoose/blog_app/assets/55840999/c0e60b7f-738b-47b3-ad7d-6cdb2aa2c125)|